packages/flutter/test/widgets/basic_test.dart

```diff
@@ -54,6 +54,37 @@ void main() {
 
   });
 
+  group('FractionalTranslation', () {
+    testWidgets('hit test', (WidgetTester tester) async {
+      final GlobalKey key1 = new GlobalKey();
+      bool _pointerDown = false;
+
+      await tester.pumpWidget(
+        new Center(
+          child: new FractionalTranslation(
+            translation: const Offset(1.0, 1.0),
```

Comment:
@Hixie This test seems legit to me, but it's failing if we override `RenderFractionalTranslation.hitTestChildren` instead of `RenderFractionalTranslation.hitTest`. Does this seem suspicious?

Reply:
We probably want the same hack as for RenderTransform, where we skip the usual hitTest logic.

What's going on is that the key1 box is entirely outside (offset by 100% of the height and width) of the FractionalTranslation widget. So when we do the hit testing for the FractionalTranslation, we find that the hit is not at all inside that widget, and we therefore fail the hit test, without even checking with the child.

Reply:
I see. That should be fixed now. I also added two more tests for entirely/partially inside the bounding box.

For `RenderFollowerLayer`, it appears to be tested here: https://github.com/flutter/flutter/blob/c46347b6df27abdb91b8b7c63e90255fefc86a6e/packages/flutter/test/widgets/composited_transform_test.dart#L140